mfhdf/dumper/hdp.c

```diff
@@ -148,36 +169,37 @@ main(int argc, char *argv[])
                           fprintf( stderr, ">>> Please make a note that dumprig is no longer available.\n");
                           fprintf( stderr, "    The command dumpgr is and should be used in its place.\n" );
                           if (FAIL == do_dumpgr(curr_arg, argc, argv, glob_opts.help)) */
-                exit(1);
+                exit(EXIT_FAILURE);
             break;
 
         case DUMPVG:
             if (FAIL == do_dumpvg(curr_arg, argc, argv, glob_opts.help))
-                exit(1);
+                exit(EXIT_FAILURE);
             break;
 
         case DUMPVD:
             if (FAIL == do_dumpvd(curr_arg, argc, argv, glob_opts.help))
-                exit(1);
+                exit(EXIT_FAILURE);
             break;
 
         case DUMPGR:
             if (FAIL == do_dumpgr(curr_arg, argc, argv, glob_opts.help))
-                exit(1);
+                exit(EXIT_FAILURE);
             break;
 
         case HELP:
-        case NONE:
             usage(argc, argv);
             break;
 
+        /* Should not get here - invalid commands are handled earlier */
+        case BAD_COMMAND:
         default:
-            printf("Invalid command!, cmd=%d\n", (int)cmd);
-            exit(1);
+            printf("Invalid command!\n");
+            exit(EXIT_FAILURE);
             break;
-    } /* end switch */
```

Comment:
Printing the enum value was unhelpful. Printing the string is better, which we do above.